tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,71 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    ceph_version = yaml.safe_load(response.text)[5.1]["ceph-version"]
+    return ceph_version
+
+
+def run(ceph_cluster, **kw):
+    """
+    Args: Check the latest image version using cephadm pull option
+        **kw: Key/value pairs of configuration information to be used in the test.
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    Test case covered : CEPH-83573787
+    Test Case Flow:
+    1. Deploy a 5.x cluster
+    2. check for container image latest available using cephadm pull option.
+    """
+
+    try:
+        config = kw.get("config")
+
+        log.info("Executing cephadm pull command")
+        instance = CephAdmin(cluster=ceph_cluster, **config)
+
+        # Geting ceph version detail using cephadm pull command
+        out, _ = instance.installer.exec_command(cmd="cephadm pull", sudo=True)
+
+        ceph_version_cephadm = ast.literal_eval(out)["ceph_version"].split(" ")[2][:-6]
+
+        # Geting ceph version detail from magna002 latest ceph container yaml
+        ceph_version = getting_ceph_version(
+            "http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/release.yaml"
+        ).split(" ")[-1]
+        log.info(
+            "cepadm pulled ceph version: %s\nCeph version available in ceph container yaml: %s"
```

Comment:
It should `cephadm` not `cepadm` Please correct that mistake.

Reply:
Thanks @sunilangadi2 
Updated